### PR TITLE
Dzelge xxx gen2 no traceback

### PR DESCRIPTION
**Pull request by dzelge**
When accepting this PR xcube gen2 will print tracebacks to stderr when raising  CubeGeneratorErrors (#448)

EDIT: No unit tests for xcube/cli/gen2.py

[Description of PR]

Checklist:

* [x] Add unit tests and/or doctests in docstrings
* [x] Add docstrings and API docs for any new/modified user-facing classes and functions
* [x] New/modified features documented in `docs/source/*`
* [ x ] Changes documented in `docs/CHANGES.md`
* [ ] AppVeyor and Travis CI passes
* [x] Test coverage remains or increases (target 100%)
* [ ] Associated issues closed after merge

### CODE REVIEW

**Review by pont-us — APPROVED** [review]
Looks fine. Twelve unit tests are failing in CI, but these failures are caused (I'm fairly sure) by the xarray 0.18.0 release yesterday, and are also failing in master.

**Review by TonioF — APPROVED** [review]
Looks good, but I really wish we had a test for this.

File: xcube/cli/gen2.py [hunk 1]
```diff
@@ -135,6 +135,7 @@ def dump_cube_info(cube_info: CubeInfo):
             print(**print_kwargs)
             print('Remote traceback:', **print_kwargs)
             print('=================', **print_kwargs)
+            print(e.remote_traceback, file=sys.stderr)
```
Comment (forman) [inline xcube/cli/gen2.py:138]:
```suggestion
            print(e.remote_traceback, **print_kwargs)
```

**Review by forman — CHANGES_REQUESTED** [review]
Tiny change only.

**Review by forman — CHANGES_REQUESTED** [review]
Please
- Fix failing AppVeyor build
- Resolve conflicts
- Use checklist

**Review by forman — APPROVED** [review]

**Pull request merged**